src/diffusers/utils/dduf.py

```diff
@@ -0,0 +1,41 @@
+import zipfile
+
+
+class DDUFReader:
```

Comment:
TODO:

Take this from https://github.com/huggingface/huggingface_hub/pull/2692/

Reply:
switched to huggingface hub api